### PR TITLE
Event UI

### PR DESCRIPTION
**Pull request by Eliasgoat**
User interface of an event from the point of view of a general user. The register button is not functional yet.

### CODE REVIEW

File: app/src/androidTest/java/com/github/se/gatherspot/UserFirebaseConnectionUnit.kt
Comment (SamBachmann) [inline app/src/androidTest/java/com/github/se/gatherspot/UserFirebaseConnectionUnit.kt:1]: Looks good to replace Intereset set by a list

File: app/src/main/java/com/github/se/gatherspot/ui/EventUI.kt [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+                  }
+                  event.timeEnding?.let { endTime ->
+                    Text(endTime.format(DateTimeFormatter.ofLocalizedTime(FormatStyle.SHORT)))
+                  }
+                }
+              }
+
+              // Inscription Limit Date and Time
+              Spacer(modifier = Modifier.height(8.dp))
+              Text(
```
Comment (SamBachmann) [inline app/src/main/java/com/github/se/gatherspot/ui/EventUI.kt:169]: Inscription date and time limits are optionals. What if there are not specified ?

**Review by SamBachmann — APPROVED** [review]
Good work, clean interface and well suited testing

**Review by SamBachmann — APPROVED** [review]
Everything is ok

File: app/src/main/java/com/github/se/gatherspot/ui/EventUI.kt (hunk 1 above)
Comment (Eliasgoat) [reply app/src/main/java/com/github/se/gatherspot/ui/EventUI.kt:169]: The question mark after the inscription date shows that the text will not be showed if the date is null.

**Pull request merged**